packages/block-editor/src/components/inserter/hooks/use-patterns-state.js

```diff
@@ -68,7 +80,7 @@ const usePatternsState = ( onInsert, rootClientId ) => {
 		[ createSuccessNotice, onInsert ]
 	);
 
-	return [ patterns, allCategories, onClickPattern ];
+	return { patterns, allCategories, onClickPattern };
```

Comment:
Was this change necessary at all? Now it's not consistent anymore with useBlocksState.

Reply:
The object return makes for a nicer API, eg. avoids the likes of `const [ patterns, , onClickPattern ] = `. An array return makes sense for hooks like useState where you are always going to use both values and want to easily give them context-specific names, but not so much in instances like this hook. 

I don't have a strong opinion on this though. If keeping it consistent with `useBlocksState` is thought to be more important then there is a [revert PR here](https://github.com/WordPress/gutenberg/pull/54582).